MSAL/src/public/MSALPublicClientApplication.h

```diff
@@ -43,45 +43,45 @@
 @interface MSALPublicClientApplication : NSObject
 
 /*!
+    A copy of the configuration which was provided in the initializer.
```

Comment:
nit: add a warning here that modifying configuration after creation wouldn't change anything?

Reply:
done